.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="PROJECT_PROFILE" value="Project Default" />
+    <option name="USE_PROJECT_PROFILE" value="true" />
+    <version value="1.0" />
+  </settings>
+</component>
```

Comment:
The .idea files are just configuration for the dev environment, should be removed and .idea added to the gitignore file.